bin/jsoo_main.ml

```diff
@@ -111,16 +107,19 @@ let export (field : string) v = Js.Unsafe.set Js.Unsafe.global field v
 
 let () = Load_path.add_dir "/static"
 
-let make_compiler name impl =
-  export name
+let () =
+  export "ocaml"
```

Comment:
perhaps we should call this `melange`.

feel free to do in a future PR, I don't want to unblock this one further